package.json

```diff
@@ -47,6 +48,7 @@
     "tiptap-extensions": "^1.33.1",
     "v-tooltip": "akryum/v-tooltip",
     "vue": "^2.6.10",
+    "vue-chartjs": "^3.5.1",
```

Comment:
Do we need to list chart.js as well as this library?

Reply:
https://github.com/chec/ui-library/pull/429/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R33

Reply:
Is this a link back to this comment? I'm confused

Reply:
I see you've highlighted a line - my question is whether you need to specify that line, since you're specifying vue-chartjs which presumably lists chart.js as a dependency already

Reply:
Oh sorry, I thought you were asking if I needed to add chart.js. Chart.js is listed as a peer dependency in `vue-chartjs` so yeah, I need `chart.js` in the package.json too.